ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessor.kt

```diff
@@ -7,6 +7,11 @@ import java.util.Locale
 internal object VariableProcessor {
     private val REGEX = Regex("\\{\\{\\s[a-zA-Z0-9_]+\\s\\}\\}")
 
+    /**
+     * Information necessary for computing variables.
+     */
+    data class Context(val discountRelativeToMostExpensivePerMonth: Double?)
```

Comment:
I'm not very convinced of this naming. It made me think it had something to do with Android Context at first. Do you think you can find an alternative? Maybe `Data`?

Reply:
I agree, it's confusing in android

Reply:
Good point, renamed